sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -16,6 +27,9 @@
 
 #include <_az_cfg_prefix.h>
 
+/**
+ * @brief Client is fixed to a specific version of the Azure Blob Storage service
```

Comment:
We use `Azure Blob Storage` service here, but above we say  `Azure Storage Blob` client

Is the client ASB and service ABS?

Reply:
Should be Azure Storage Blobs client, good catch

Reply:
Sorry, let me clarify.

Isn't the service called Azure Blob Storage (swap the words storage and blog)?
https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction
https://azure.microsoft.com/en-us/services/storage/blobs/

Looks like the client/packages are called Azure Storage Blobs in other languages, but that ordering reads like a weird service name.
https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Blobs

Reply:
I followed what other languages did.  We can revisit for the next release